packages/create-sitecore-jss/src/templates/nextjs-sxa/src/components/Title.tsx

```diff
@@ -17,7 +17,8 @@ interface Fields {
       field: {
         jsonValue: {
           value: string;
-          editable: string;
+          editable?: string;
+          metadata?: object;
```

Comment:
Let's make it consistent
```suggestion
          metadata?: { [key: string]: unknown };
```